README.md

```diff
@@ -42,5 +44,5 @@ $ add-dependencies moment@2.0.0 react@16.8 redux eslint --dev
 or with `npx`:
 
 ```sh
-$ npx add-dependencies moment@2.0.0 react@16.8 redux eslint --dev
+$ npx add-dependencies ./package.json moment@2.0.0 react@16.8 redux eslint --dev
```

Comment:
This example is irrelevant, we don't need to use `./package.json` here since it's the default value. In the example you'd better use some file path like `/home/user/project/package.json`.

Reply:
addressed